src/components/vm/vmCloneDialog.tsx

```diff
@@ -57,10 +74,16 @@ export const CloneDialog = ({ name, connectionName }) => {
         }
 
         setInProgress(true);
-        const options = { pty: true };
-        if (connectionName === "system")
-            options.superuser = "try";
-        return cockpit.spawn(["virt-clone", "--connect", "qemu:///" + connectionName, "--original", name, "--name", newVmName, "--auto-clone"], options)
+        return cockpit.spawn(
+            [
+                "virt-clone", "--connect", "qemu:///" + connectionName,
+                "--original", name, "--name", newVmName,
+                "--auto-clone"
+            ],
+            {
+                pty: true,
+                ...(connectionName === "system" ? { superuser: "try" } : { })
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2187-561408b0-20250717-145115-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)